src/ngamsServer/ngamsServer/commands/register.py

```diff
@@ -576,7 +576,7 @@ def register(srvObj,
                  NGAMS_VOLUME_ID_FILE, \
                  NGAMS_VOLUME_INFO_FILE]:
                     mimeType = ngamsLib.detMimeType(mimeTypeMappings,
-                                                    nextFile, 0)
+                                                    nextFile, 1)
```

Comment:
At first I thought this wasn't a good idea, but then I delved into `detMimeType` and saw it returns `unknown` when one instructs it to not throw an exception. And as long as there is a registration plug-in for that type (in your tests there is) then it should be fine.

Reply:
In the original comment I mentioned that it is better to let the plugin decide what to do with files that have unknown mime-types. We need this for the ALMA plugin so it can then parse the MIME data file which has no identifying extension. IN the generic plugin I throw the original exception in the event the mime-type is `unknown` so that it still maintains the original behavior.